### PR TITLE
added: Doc: monika logo in readme #39

### PR DESCRIPTION
**Pull request by nicnocquee**
This PR adds Monika logo to Readme. The logo is from the landing page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
...TEuMzA3IDkyLjcxMzQgMTAuOTM4MkM5My4wNjE3IDEwLjU0OSA5My40ODE2IDEwLjI1MiA5My45NzMyIDEwLjA0NzJDOTQuNDg1MyA5LjgyMTgyIDk1LjA2OTIgOS43MDkxNSA5NS43MjQ3IDkuNzA5MTVDOTYuMzU5NyA5LjcwOTE1IDk2LjkyMyA5LjgyMTgyIDk3LjQxNDcgMTAuMDQ3MkM5Ny45MjY4IDEwLjI3MjUgOTguMzU3IDEwLjU3OTggOTguNzA1MiAxMC45NjlDOTkuMDUzNSAxMS4zNTgyIDk5LjMxOTcgMTEuODA4OCA5OS41MDQxIDEyLjMyMUM5OS42ODg1IDEyLjgxMjYgOTkuNzgwNyAxMy4zMzUgOTkuNzgwNyAxMy44ODhDOTkuNzgwNyAxNC40NDExIDk5LjY3ODIgMTQuOTczNyA5OS40NzM0IDE1LjQ4NTlDOTkuMjg5IDE1Ljk3NzUgOTkuMDIyNyAxNi40MTc5IDk4LjY3NDUgMTYuODA3MUM5OC4zMjYyIDE3LjE3NTggOTcuODk2MSAxNy40NzI5IDk3LjM4MzkgMTcuNjk4MkM5Ni44OTIzIDE3LjkyMzUgOTYuMzM5MiAxOC4wMzYyIDk1LjcyNDcgMTguMDM2MkM5NS4wNDg3IDE4LjAzNjIgOTQuNDU0NiAxNy45MjM1IDkzLjk0MjUgMTcuNjk4MkM5My40MzA0IDE3LjQ3MjkgOTMuMDEwNCAxNy4xNzU4IDkyLjY4MjcgMTYuODA3MUM5Mi4zNTQ5IDE2LjQxNzkgOTIuMTA5MSAxNS45Nzc1IDkxLjk0NTIgMTUuNDg1OVoiIGZpbGw9InVybCgjcGFpbnQwX2xpbmVhcikiLz4KPGRlZnM+CjxsaW5lYXJHcmFkaWVudCBpZD0icGFpbnQwX2xpbmVhciIgeDE9IjUuNTAwMTYiIHkxPSIyMi4wMDA2IiB4Mj0iMTA4LjYyOCIgeTI9Ii05LjYyNTI2IiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSI+CjxzdG9wIHN0b3AtY29sb3I9IiM5ODdDRTgiLz4KPHN0b3Agb2Zmc2V0PSIxIiBzdG9wLWNvbG9yPSIjMkZEQ0RDIi8+CjwvbGluZWFyR3JhZGllbnQ+CjwvZGVmcz4KPC9zdmc+Cg==)
```
Comment (samsunoor) [inline README.md:1]: I guess github renderer does not work this way? [preview reference](https://github.com/hyperjumptech/monika/blob/13f94fa6ccef213f1e6331f9a509c17166746e4c/README.md)

Comment (dennypradipta) [reply README.md:1]: :+1:

Comment (nicnocquee) [reply README.md:1]: damn, let me change to a real file then.

Comment (nicnocquee) [reply README.md]: should be fixed now.

**Review by raosan — APPROVED** [review]

**Review by dennypradipta — APPROVED** [review]

**Pull request merged**